src/renderer/features/multisig-wallet/index.tsx

```diff
@@ -170,21 +170,8 @@ transactionSDK(multisigWalletFeature, {
       const extrinsic = transactionService.createExtrinsic(transaction, api);
 
       return transactionService.getExtrinsicWeight(extrinsic).then(maxWeight => {
-        const multisigTransaction: MultisigTransaction = {
```

Comment:
Its incorrect to use `maxWeight` of original transaction for flexible multisig since it additionaly wraps original tx into proxy. So if we want to do `transfer`, then weight of `transfer` will be different (and smaller) than the weight of `proxy(transfer)` so `maxWeight` used here might be underestimate